### PR TITLE
Initial TSC governance proposal

### PR DESCRIPTION
**Pull request by balhar-jakub**
Linked to: #923 

Signed-off-by: Jakub Balhar <jakub.balhar@broadcom.com>

### CODE REVIEW

File: Technical-Steering-Committee/tsc-governance.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Initially the TSC was formed with three permanent members appointed by the ZLC. To appoint a permanent member of the TSC requires a successful vote by the TSC on such an appointment. 
+
+### Squad representatives
+
+Each squad that is part of the Zowe project has its representative on the TSC. The squad votes on the representative on a yearly basis. The details of the vote is left to the squads.
+
+### Removal from the TSC role
+
+It’s possible to remove a person from the role on the TSC. To do so two-thirds vote of the entire TSC is required. If such person was appointed the squad, the squad needs to elect another representative on next squad meeting, if the person is permanent member then the person is removed from the document and loses voting rights.
```
Comment (balhar-jakub) [inline Technical-Steering-Committee/tsc-governance.md:15]: Update the clarification to be more.

Comment (balhar-jakub) [reply Technical-Steering-Committee/tsc-governance.md:15]: Suggestions on the situations and the role of the mediations.

File: Technical-Steering-Committee/tsc-governance.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Each squad that is part of the Zowe project has its representative on the TSC. The squad votes on the representative on a yearly basis. The details of the vote is left to the squads.
+
+### Removal from the TSC role
+
+It’s possible to remove a person from the role on the TSC. To do so two-thirds vote of the entire TSC is required. If such person was appointed the squad, the squad needs to elect another representative on next squad meeting, if the person is permanent member then the person is removed from the document and loses voting rights. 
+
+## Roles
+
+The TSC has internally decided to create three roles focusing on different aspects of the governance of the Zowe. The roles are held by the TSC members and the members are voted into the roles. The [https://github.com/zowe/community/blob/master/README.md](https://github.com/zowe/community/blob/master/README.md) contains the currently appointed members with the function time frames. The time frame for helding the role is year. It’s possible for one person to hold all three roles.
```
Comment (balhar-jakub) [inline Technical-Steering-Committee/tsc-governance.md:19]: Consecutive roles also ok

**Review by RASakach — COMMENTED** [review]
Consider rewording this "Permanent members" section as follows:

Initially the TSC was formed with three permanent members appointed by the ZLC.  The process for appointing any new permanent member of the TSC requires (1) a nomination by an existing TSC member and (2) a  two-thirds vote by the  full TSC on such an appointment.

**Pull request merged**